functions/menu.bash

```diff
@@ -217,6 +218,7 @@ show_main_menu() {
     case "$choice2" in
       50\ *) backup_openhab_config ;;
       51\ *) restore_openhab_config ;;
+      *Restore\ text\ only*) restore_openhab_config "--textonly" ;;
```

Comment:
Why use GNU long style options here? Its not how we usually do it so I would prefer if we only kept the actual flag when we call the command.

Reply:
it was because that's what gets passed to the binary.
But you're right in terms of clean coding that's a fail so